src/test/java/core/basesyntax/RegistrationServiceTest.java

```diff
@@ -0,0 +1,135 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationException;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import java.util.Objects;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceTest {
+    private static final String VALID_STRING = "string";
+    private static final String ANOTHER_VALID_STRING = "another";
+    private static final String INVALID_STRING_WITH_LENGTH_ZERO = "";
+    private static final String INVALID_STRING_WITH_LENGTH_THREE = "qwe";
+    private static final String INVALID_STRING_WITH_LENGTH_FIVE = "asdfg";
```

Comment:
use meaningful constants. if you want to have valid login - you should have VALID_LOGIN constant and so on.